docs/locale.mdx

```diff
@@ -0,0 +1,181 @@
+---
+sidebar_position: 3
+title: Translations and Custom Messages
+---
+
+EssentialsX includes messages for a wide range of languages out-of-the-box, as well as extensive support for customising
```

Comment:
Change "Customising" to "Customizing"  - Yes i know that the current locale page us UK standard, but both are used, and we need to stick to a standard.